src/test/java/com/cgdpd/library/service/BookServiceTest.java

```diff
@@ -86,4 +98,45 @@ public void should_throw_exception_if_author_not_exist() {
 
         verifyNoInteractions(bookRepository);
     }
+
+    @Test
+    public void should_return_books_and_pages() {
```

Comment:
Read the test out loud and check if it makes sense.
Is it returning books and pages or paginated books?
Test names should be accurate and explanatory 
```suggestion
    public void should_return_books_with_pagination() {
```

Also create multiple tests to check for different scenarios
- Return last page available
- Check for when we receive invalid input by requesting page that doesn't exist